python/embag.cc

```diff
@@ -77,6 +90,9 @@ PYBIND11_MODULE(libembag, m) {
       .def("data", [](std::shared_ptr<Embag::RosMessage> &m) {
         return m->data();
       })
+      .def_property_readonly("raw_data", [](std::shared_ptr<Embag::RosMessage> &m) {
+        return py::bytes(&m->raw_buffer->at(m->raw_buffer_offset), m->raw_data_len);
+      })
```

Comment:
Do we also want to add `raw=True` functionality to the `read_messages` interface in `embag.Bag`?

Reply:
I could - but to do that (and maintain compatibility with rosbag) I would need to change the constructor of RosMessage and add a new field for the raw message definition.

Reply:
Going to leave this for potential future improvement!